reuters/dataset_infos.json

```diff
@@ -0,0 +1 @@
n,pages = \"212--217\"\n}\n\n@inproceedings{LEWIS94b\n ,author = \"David D. Lewis and Marc Ringuette\"\n ,title = \"A Comparison of Two Learning Algorithms for Text Categorization\"\n ,booktitle = \"Symposium on Document Analysis and Information Retrieval\"\n ,year = 1994\n ,organization = \"ISRI; Univ. of Nevada, Las Vegas\"\n ,address = \"Las Vegas, NV\"\n ,month = apr\n ,pages = \"81--93\"\n}\n\n@article{LEWIS94d\n, author       = \"David D. Lewis and Philip J. Hayes\"\n, title        = \"Guest Editorial\"\n, journal      = \"ACM Transactions on Information Systems\"\n, year         = 1994 \n, volume       = 12\n, number       = 3\n, pages        = \"231\"\n, month        = jul\n}\n\n@article{SPARCKJONES76\n,author = \"K. {Sparck Jones} and  C. J. {van Rijsbergen}\"\n,title =  \"Information Retrieval Test Collections\"\n,journal = \"Journal of Documentation\"\n,year = 1976\n,volume = 32\n,number = 1\n,pages = \"59--75\"\n  }\n\n@book{WEISS91\n ,author = \"Sholom M. Weiss and Casimir A. Kulikowski\"\n ,title = \"Computer Systems That Learn\" \n ,publisher = \"Morgan Kaufmann\"\n ,year = 1991\n ,address = \"San Mateo, CA\"\n }\n", "homepage": "https://kdd.ics.uci.edu/databases/reuters21578/reuters21578.html", "license": "", "features": {"text": {"dtype": "string", "id": null, "_type": "Value"}, "topics": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "lewis_split": {"dtype": "string", "id": null, "_type": "Value"}, "cgis_split": {"dtype": "string", "id": null, "_type": "Value"}, "old_id": {"dtype": "string", "id": null, "_type": "Value"}, "new_id": {"dtype": "string", "id": null, "_type": "Value"}, "places": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "people": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "orgs": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "exchanges": {"feature": {"dtype": "string", "id": null, "_type": "Value"}, "length": -1, "id": null, "_type": "Sequence"}, "date": {"dtype": "string", "id": null, "_type": "Value"}, "title": {"dtype": "string", "id": null, "_type": "Value"}}, "supervised_keys": null, "builder_name": "reuters21578", "config_name": "default", "version": {"version_str": "0.0.0", "description": null, "nlp_version_to_prepare": null, "major": 0, "minor": 0, "patch": 0}, "splits": {"test": {"name": "test", "num_bytes": 930193, "num_examples": 720, "dataset_name": "reuters21578"}, "train": {"name": "train", "num_bytes": 18050316, "num_examples": 18323, "dataset_name": "reuters21578"}}, "download_checksums": {"https://kdd.ics.uci.edu/databases/reuters21578/reuters21578.tar.gz": {"num_bytes": 8150596, "checksum": "3bae43c9b14e387f76a61b6d82bf98a4fb5d3ef99ef7e7075ff2ccbcf59f9d30"}}, "download_size": 8150596, "dataset_size": 18980509, "size_in_bytes": 27131105}}
```

Comment:
I guess this file and `reuters/dummy/0.0.0/dummy_data.zip` are supposed to be removed ?